### PR TITLE
Escaping searchQuery for $regex match

### PR DESCRIPTION
**Pull request by Akarshit**
Used [escape-string-regexp](https://www.npmjs.com/package/escape-string-regexp)(one line of code and half the no. of downloads of lodash!) to escape the `searchQuery`
### To test
1. Create an order with an email with a plus sign in it
2. Go to the new order search in the dashboard
3. Search for this email
4. The order will show up

Please note this PR makes changes to `package.json`

### CODE REVIEW

File: imports/plugins/included/search-mongo/server/publications/searchresults.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Roles } from "meteor/alanning:roles";
 import { check, Match } from "meteor/check";
 import { Reaction, Logger } from "/server/api";
 import { ProductSearch, OrderSearch, AccountSearch } from "/lib/collections";
+import escapeStringRegex from 'escape-string-regexp';
```
Comment (zenweasel) [inline imports/plugins/included/search-mongo/server/publications/searchresults.js]: Import order (NPM packages go first)

**Comment by zenweasel** [comment]
@Akarshit When pulling this branch to test I can't fast-forward. I think you need to update your branch to the current marketplace.

**Comment by Akarshit** [comment]
@zenweasel Working on it.

**Comment by spencern** [comment]
FYI @Akarshit, I've added you as a collaborator on our repo, so you should be able to create a branch and push your branch directly to the reactioncommerce/reaction repo which will permit developers on our team to update PRs you make and streamline the PR review process. Probably don't need to change this one, but worth doing for your next PR.

Thanks again for the contribution.

**Comment by Akarshit** [comment]
@spencern Cool! Will keep that in mind from now on.

**Comment by zenweasel** [comment]
Tested. Verified fixed.

**Pull request merged**